src/mlang/m_frontend/mast.ml

```diff
@@ -98,8 +98,9 @@ type set_value =
 
 (** Values that can be substituted for loop parameters *)
 type set_value_loop =
-  | VarParam of variable_name Pos.marked
-  | IntervalLoop of literal Pos.marked * literal Pos.marked
+  | Single of literal Pos.marked
+  | Range of literal Pos.marked * literal Pos.marked
+  | NumRange of literal Pos.marked * literal Pos.marked
```

Comment:
Could we change `NumRange` into `Interval`? I think it would make clearer that it expects to numbers. Contrary to Range where it can either be numbers or names.

Reply:
Sure, I'll do that.